music/src/core/player.ts

```diff
@@ -140,6 +157,7 @@ export abstract class BasePlayer {
         }, t);
       }
     }, seq.notes.map(n => [n.startTime, n]));
+    Tone.Transport.bpm.value = this.desiredBPM;
```

Comment:
If a new qpm is passed in the arguments, does that become the desired qpm?

Reply:
ah good catch, fixed.